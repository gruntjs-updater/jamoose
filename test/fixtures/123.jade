html
  head
    link(rel="stylesheet", href="test/fixtures/css/test.css")

  body
    h1 {{name}}
    ol
      li one
      li two
      li three