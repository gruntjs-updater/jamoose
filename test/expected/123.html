<html><head></head><body><h1>{{name}}</h1><ol style="list-style: none;"><li style="color: green;">one</li><li style="color: green;">two</li><li style="color: green;">three</li></ol></body></html>
